apps/monitoring/check_methods.py

```diff
@@ -1,76 +0,0 @@
-import os
-from ftplib import FTP
-
-import requests
-
-from apps.plea.models import Case
-
-
-__all__ = ("check_url", "check_ftp", "check_address", "check_mandrill", "check_database")
-
-
-def check_url(url, timeout=3):
-    """
-    Verify that a url exists and returns a 200 status
-    in a reasonable time period.
-    """
-
-    try:
-        response = requests.get(url, timeout=timeout, verify=False)
-    except requests.exceptions.RequestException:
-        return False
-    else:
-        if response.status_code != 200:
-            return False
-
-    return True
-
-
-def check_ftp(host, port, timeout=3):
-    """
-    Verify that we can make a connection to an FTP server
-    """
-
-    ftp = FTP()
-
-    try:
-        ftp.connect(host, port, timeout)
-
-    except Exception:
-        return False
-    else:
-        ftp.close()
-
-    return True
-
-
-def check_address(address):
-    """
-    Check that an address can be ICMP pinged
-    """
-
-    if os.system("ping -c 1 {}".format(address)) == 0:
-        return True
```

Comment:
I'm not sure - I probably found an example that did it that way.  But that's old ping/healthcode code which was redundant.  It was replaced by the moj-irat library sometime ago. So I removed that file and used the same app for the new service monitoring code.